examples/index.html

```diff
@@ -51,7 +51,9 @@ <h1>Demo pages</h1>
 				<pfe-cta pfe-priority="secondary" pfe-variant="wind"><a href="../elements/pfe-avatar/demo">pfe-avatar</a></pfe-cta>
 				<pfe-cta pfe-priority="secondary" pfe-variant="wind"><a href="../elements/pfe-badge/demo">pfe-badge</a></pfe-cta>
 				<pfe-cta pfe-priority="secondary" pfe-variant="wind"><a href="../elements/pfe-band/demo">pfe-band</a></pfe-cta>
+				<pfe-cta pfe-priority="secondary" pfe-variant="wind"><a href="../elements/pfe-button/demo">pfe-button</a></pfe-cta>
 				<pfe-cta pfe-priority="secondary" pfe-variant="wind"><a href="../elements/pfe-card/demo">pfe-card</a></pfe-cta>
+				<pfe-cta pfe-priority="secondary" pfe-variant="wind"><a href="../elements/pfe-chip/demo">pfe-chip</a></pfe-cta>
```

Comment:
I would remove the pfe-chip and pfe-switch folders and rebuild so that this index file is clean